### DISCUSSION TITLE
New git clone question // follow-up non-question

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#890] [Q&A] [answered]
If I wanted to delete the automatic folder and do a brand new `git clone`, which folders do I keep to prevent long/large downloads?

I know that `outputs`, `models` and `extensions` are folders I would want to keep, are there any other largish folders that won't affect a new cloning?

**Comment by vladmandic** [comment, marked as answer]
you can also keep repositories if you really want, that's going to save you ~900mb.

**Comment by brknsoul** [comment]
I did a `git reset --hard`, which killed my extensions folder (probably shouldn't happen), but kept my models folder intact (thank goodness!).

I think I messed up originally installing this, pointing it to an old directml branch that's now been incorporated in the master branch.

I'll try keeping the `repositories` folder as well.

**Comment by brknsoul** [comment]
Holy crap.. just reinstalled sd.next and now I'm getting it/s instead of s/it! Granted it's only ~1.40 it/s but for windows/directml, that's a definite step forward!
